packages/build-utils/src/monorepos/monorepo-managers.ts

```diff
@@ -0,0 +1,53 @@
+import type { Framework } from '@vercel/frameworks';
+
+/**
+ * The supported list of monorepo managers.
+ *
+ * This list is designed to work with the @see {@link detectFramework} function.
+ *
+ * @example
+ *   import { monorepoManagers as frameworkList } from '@vercel/build-utils'
+ *   import { detectFramework } from '@vercel/build-utils'
+ *
+ *   const fs = new GitDetectorFilesystem(...)
+ *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
+ *
+ * @todo Will be used by the detect-eligible-projects API endpoint for a given git url.
+ */
+export const monorepoManagers: Array<Framework> = [
```

Comment:
Should this code live in `@vercel/frameworks` instead?

Thoughts @TooTallNate ?

Reply:
Realistically there are three things at play: frameworks (lol no way to list them all), package managers (npm, pnpm, yarn, berry), package manager managers (Turborepo, lerna, lerna-lite, [wireit](https://github.com/google/wireit), ...)

Note that I'd *maybe* shy away from the term "monorepo" as we don't consider Turborepo to be constrained by monorepo.

Reply:
Happy to move code if needed.  Our thoughts were to just add them to `build-utils` at first, then refactor if needed

Reply:
could call the new package (if needed) `@vercel/repo-utils`

Reply:
Yeah, this is just a naming/organization question; I have no strong opinions other than avoiding the word "monorepo." And maybe even "repo" ... `@vercel/project-utils` maybe?